src/main/java/racingcar/game/Cars.java

```diff
@@ -0,0 +1,63 @@
+package racingcar.game;
+
+import racingcar.controller.Dice;
+import racingcar.view.OutputView;
+import static racingcar.controller.constants.IntegerConstants.*;
+import static racingcar.controller.constants.DelimiterConstants.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    ArrayList<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
    private final ArrayList<Car> cars = new ArrayList<>();
```
encapsulation에 대해서 신경 써주시는 것은 어떨까 싶습니다,,! 🤗
https://jojoldu.tistory.com/412